lib/compiler.js

```diff
@@ -68,9 +70,27 @@ exports.compile = function (code, options) {
   }
 
   result.code = magicString.toString();
+
   return result;
 };
 
 exports.transform = function (ast, options) {
   return dynRequire("./transform.js")(ast, options);
 };
+
+function makeUniqueIdQuickly(prefix, source) {
+  const scanRegExp = new RegExp("\\b" + prefix + "(\\d*)\\b", "g");
```

Comment:
No need to tweak anything yet, but I'll look at benchmarking this vs using a canned regexp for the common case of "module".